generate/generate.go

```diff
@@ -39,180 +39,17 @@ func New() Generator {
 			OS:   runtime.GOOS,
 			Arch: runtime.GOARCH,
 		},
-		Root: rspec.Root{
-			Path:     "",
```

Comment:
This is a [REQUIRED property][1], so I think you should keep this.  If you don't like the empty-string value, I'd be ok with a `"FIXME"` default.

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc5/config.md#root